README.md

```diff
@@ -77,6 +79,36 @@ If you're working with multi-page documents, append the index of the page as an
 
 The plugin will export the diagram to the `format` specified in your configuration and will rewrite the `<img>` tag in the generated page to match. To speed up your documentation rebuilds, the generated output will be placed into `cache_dir` and then copied to the desired destination. The cached images will only be updated if the source diagram's modification date is newer than the cached export. Thus, bear in mind caching works per file - with large multi-page documents a change to one page will rebuild all pages, which will be slower than separate files per page.
 
+### Linking draw.io colours with theme CSS
+
+If you want to link the colours in your draw.io diagrams with your MkDocs theme CSS, set `link_css` to `true` in your configuration. This will replace the colours in your diagrams with the colours defined in your extra CSS. This is useful if you want to ensure your diagrams match the look and feel of your documentation, especially when toggling between light and dark themes.
```

Comment:
Rename option as above, rephrase and call out `extra_css` by name:

```suggestion
If you want to source colors in your diagrams from your MkDocs theme or custom CSS, set `use_theme_colors` to `true` in your configuration. This will replace the colors in your diagrams with the colours defined in your theme or `extra_css`. This is useful if you want to ensure your diagrams match the look and feel of your documentation, especially when toggling between light and dark themes.
```